src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -13,12 +13,22 @@
 
 namespace Neo.SmartContract.Native
 {
+    /// <summary>
+    /// Represents the NEO token in the NEO system.
+    /// </summary>
     public sealed class NeoToken : FungibleToken<NeoToken.NeoAccountState>
     {
         public override string Symbol => "NEO";
         public override byte Decimals => 0;
+
+        /// <summary>
+        /// Indicates the total amount of NEO.
+        /// </summary>
         public BigInteger TotalAmount { get; }
 
+        /// <summary>
+        /// Indicates the effective voting turnout in NEO. Voting is valid only when the voting turnout exceeds this value.
```

Comment:
"The voted candidates will only be effective when ....."
The consensus committee will only be defined by voting, instead of the default protocol settings, when the voting turnout exceeds this value.

Reply:
Something like that.

Reply:
Updated.